app/src/androidTest/java/com/nervousfish/nervousfish/test/LoginActivitySteps.java

```diff
@@ -56,7 +56,6 @@ public void iHaveALoginActivity() {
         intent.putExtra(ConstantKeywords.SERVICE_LOCATOR,
                 getCurrentActivity().getIntent().getSerializableExtra(ConstantKeywords.SERVICE_LOCATOR));
         getActivity().startActivity(intent);
-        assertTrue(getCurrentActivity() instanceof LoginActivity);
```

Comment:
I disagree. If you accept that as a given, you can never be sure that the next step (which depends on that activity being active) will succeed, since it can be that the activity isn't loaded yet